include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_merge_sort.h

```diff
@@ -240,6 +240,9 @@ struct __merge_sort_global_submitter<_IndexT, __internal::__optional_kernel_name
   private:
     using _merge_split_point_t = _split_point_t<_IndexT>;
 
+    // 1 final base diagonal for save final sp(0,0)
+    static constexpr std::size_t __1_final_base_diag = 1;
+
```

Comment:
If you want to have this to be a "named" literal, that is ok, but can we add it to the `__merge_sort_global_submitter` function below rather than very far away?

Reply:
Fixed